src/plugins/intel_gna/src/gna_plugin_config.cpp

```diff
@@ -313,6 +356,9 @@ void Config::AdjustKeyMapValues() {
     } else {
         keyConfigMap[GNA_CONFIG_KEY(PRECISION)] = gnaPrecision.name();
     }
+    if (execution_mode != ov::hint::ExecutionMode::UNDEFINED) {
```

Comment:
let's avoid usage of UNDEFINED
it will be removed here https://github.com/openvinotoolkit/openvino/pull/16563

Reply:
Ok, removed use of ExecutionMode::UNDEFINED.